src/factory.ts

```diff
@@ -86,6 +94,78 @@ export function buildDependencies(
     return output
 }
 
+/**
+ * Add transfer units to remove byproducts from industry outputs
+ * @param factory the FactoryGraph
+ */
+export function handleByproducts(factory: FactoryGraph) {
+    /* Loop over all factory containers */
+    for (const container of factory.containers) {
+        /* Ore containers have no byproducts */
+        if (isOre(container.item)) {
+            continue
+        }
+        const recipe = findRecipe(container.item)
+        for (const byproduct of recipe.byproducts) {
+            /* Skip ores (but there should be none anyway) */
```

Comment:
Currently hydrogen and oxygen are ores.

Reply:
Nice catch. Fixed in https://github.com/lgfrbcsgo/du-factory-generator/commit/b68fb5911d51113ba7f1700a7a570d463a554085